src/types/pill-category.ts

```diff
@@ -1,12 +1,13 @@
 export const PillCategoryValues = [
+    /* order determines how categories appear in topic landing pages */
+    'News & Announcements',
```

Comment:
News and Announcements should be on the top in topic landing pages card section.